src/input/src/simulations/logging.rs

```diff
@@ -31,12 +31,12 @@ impl Encode for LogEncoder {
             LogLevel::Debug => try!(write!(out, "[debug] ")),
             LogLevel::Info => {},
             LogLevel::Warn => {
-                try!(out.set_style(&Style::new().text(Color::Red)));
+                try!(out.set_style(Style::new().text(Color::Red)));
```

Comment:
Yes, because `text` returns a reference to self. This was sending a warning about borrowing that is insta-deref by the the compiler